etc/kayobe/inspector.yml

```diff
@@ -120,6 +120,18 @@ inspector_rules_extra:
     - action: "set-attribute"
       path: "extra/system_vendor/bios_version"
       value: "2.6.0"
+  - description: "Set the expected firmware version for Mellanox ConnectX-4"
+    conditions:
+    - field: "data://inventory.interfaces[*].product"
+      op: "matches"
+      value: "^(0x1013)$"
+    - field: "data://inventory.interfaces[*].vendor"
+      op: "matches"
+      value: "^(0x15B3)$"
+    actions:
+      - action: "extend-attribute"
+      - path: "extra/nic_firmware"
+      - value: '[{"vendor_id": "15B3", "device_id": "1013", "firmware_version": "12.20.1010"}]'
```

Comment:
This should be a single list item:

```
- action:
  path:
  value:
```

Reply:
Thanks, I was basing it the JSON examples in https://docs.openstack.org/ironic-inspector/latest/user/usage.html, where even a single action is placed in an array.